task/05-regex-tasks.js

```diff
@@ -116,14 +111,14 @@ function getRegexForSSN() {
  *   'Pa55'.match(validator) => false
  */
 function getPasswordValidator(minLength) {
-   throw new Error('Not implemented');
+  let reg = '^(?=.*[a-z])(?=.*[A-Z])(?=.*\\d)[A-Za-z\\d]' + `{${minLength},}$`;
+  return new RegExp(reg);
 }
 
-
 module.exports = {
-    getRegexForGuid: getRegexForGuid,
-    getRegexForPitSpot: getRegexForPitSpot,
-    getRegexForIPv4: getRegexForIPv4,
-    getRegexForSSN: getRegexForSSN,
-    getPasswordValidator: getPasswordValidator
+  getRegexForGuid: getRegexForGuid,
```

Comment:
the PR has redundant changes in styling, spaces, comments....
Please follow the simple rules in the future:
 - do not add any non logic changes to code if review is required
 - if the changes in styling are important - you can do it as separate PR with styling only changes to avoid noise in the main PR